ingestion/src/metadata/utils/fqn.py

```diff
@@ -142,19 +143,18 @@ def _(
     :param table_name: Table name
     :return:
     """
-    if not service_name or not table_name:
-        raise FQNBuildingException(
-            f"Service Name and Table Name should be informed, but got service=`{service_name}`, table=`{table_name}`"
-        )
-
-    fqn_search_string = _build(
-        service_name, database_name or "*", schema_name or "*", table_name
+    fqn_search_string = build_es_fqn_search_string(
+        database_name, schema_name, service_name, table_name
     )
 
-    es_result = metadata.es_search_from_fqn(
-        entity_type=Table,
-        fqn_search_string=fqn_search_string,
-    )
+    es_result = (
+        not skip_es_search
+        and metadata.es_search_from_fqn(
+            entity_type=Table,
+            fqn_search_string=fqn_search_string,
+        )
+    ) or None
+
```

Comment:
I like it, I am wondering if we could do 

```
es_result = metadata.es_search_from_fqn(entity_type=Table,fqn_search_string=fqn_search_string,) if not skip_es_search else None
```

I believe it gives the same results. This is very opinionated comment 😅 as it does not change the functionality itself. I just feel like it might make it more readible.

Reply:
Oh yes, you are right. It is clearer.